### DISCUSSION TITLE
Difference in scoring between Lighthouse and Pagespeed

### DISCUSSION THREAD

**Discussion by BenoneG** [GoogleChrome/lighthouse#14871] [General]
I tested my site on Lighthouse and Pagespeed one of them tells me i have negligible CLS, the other one fails me.
Can someone tell me why is this happening? What am i missing?

**Comment by adamraine** [comment]
We can't give any advice without a reproducible URL. These issues probably cover what you are seeing:

https://github.com/GoogleChrome/lighthouse/issues/6708
https://github.com/GoogleChrome/lighthouse/issues/10657

**Reply by BenoneG** [reply]
The URL i was testing is a website i am trying to improve.

https://www.babyneeds.ro/carucioare-2-in-1/

**Reply by BenoneG** [reply]
Tyvm for the reply. I read the articles. Some of the answers might help me. But i am unsure
